2013/01/24 15:53:45 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Can't connect to local MySQL server through socket '/Applications/XAMPP/xamppfiles/var/mysql/mysql.sock' (2)' in /DecidoDev/root/yii/framework/db/CDbConnection.php:381
Stack trace:
#0 /DecidoDev/root/yii/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /DecidoDev/root/yii/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /DecidoDev/root/yii/framework/base/CModule.php(387): CDbConnection->init()
#3 /DecidoDev/root/yii/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /DecidoDev/root/testdrive/protected/modules/user/controllers/YumInstallController.php(439): CModule->__get('db')
#5 /DecidoDev/root/yii/framework/web/actions/CInlineAction.php(49): YumInstallController->actionInstall()
#6 /DecidoDev/root/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#7 /DecidoDev/root/yii/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#8 /DecidoDev/root/yii/framework/web/filters/CFilter.php(40): CFilterChain->run()
#9 /DecidoDev/root/yii/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#10 /DecidoDev/root/yii/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#11 /DecidoDev/root/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#12 /DecidoDev/root/yii/framework/web/CController.php(291): CFilterChain->run()
#13 /DecidoDev/root/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 /DecidoDev/root/yii/framework/web/CWebApplication.php(282): CController->run('')
#15 /DecidoDev/root/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('user/install')
#16 /DecidoDev/root/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#17 /DecidoDev/root/testdrive/index.php(13): CApplication->run()
#18 {main}
REQUEST_URI=/testdrive/index.php?r=user/install
---
2013/01/24 15:54:15 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'testdrive'' in /DecidoDev/root/yii/framework/db/CDbConnection.php:381
Stack trace:
#0 /DecidoDev/root/yii/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /DecidoDev/root/yii/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /DecidoDev/root/yii/framework/base/CModule.php(387): CDbConnection->init()
#3 /DecidoDev/root/yii/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /DecidoDev/root/testdrive/protected/modules/user/controllers/YumInstallController.php(439): CModule->__get('db')
#5 /DecidoDev/root/yii/framework/web/actions/CInlineAction.php(49): YumInstallController->actionInstall()
#6 /DecidoDev/root/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#7 /DecidoDev/root/yii/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#8 /DecidoDev/root/yii/framework/web/filters/CFilter.php(40): CFilterChain->run()
#9 /DecidoDev/root/yii/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#10 /DecidoDev/root/yii/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#11 /DecidoDev/root/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#12 /DecidoDev/root/yii/framework/web/CController.php(291): CFilterChain->run()
#13 /DecidoDev/root/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 /DecidoDev/root/yii/framework/web/CWebApplication.php(282): CController->run('')
#15 /DecidoDev/root/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('user/install')
#16 /DecidoDev/root/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#17 /DecidoDev/root/testdrive/index.php(13): CApplication->run()
#18 {main}
REQUEST_URI=/testdrive/index.php?r=user/install
---
2013/01/24 15:54:18 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'testdrive'' in /DecidoDev/root/yii/framework/db/CDbConnection.php:381
Stack trace:
#0 /DecidoDev/root/yii/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /DecidoDev/root/yii/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /DecidoDev/root/yii/framework/base/CModule.php(387): CDbConnection->init()
#3 /DecidoDev/root/yii/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /DecidoDev/root/testdrive/protected/modules/user/controllers/YumInstallController.php(439): CModule->__get('db')
#5 /DecidoDev/root/yii/framework/web/actions/CInlineAction.php(49): YumInstallController->actionInstall()
#6 /DecidoDev/root/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#7 /DecidoDev/root/yii/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#8 /DecidoDev/root/yii/framework/web/filters/CFilter.php(40): CFilterChain->run()
#9 /DecidoDev/root/yii/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#10 /DecidoDev/root/yii/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#11 /DecidoDev/root/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#12 /DecidoDev/root/yii/framework/web/CController.php(291): CFilterChain->run()
#13 /DecidoDev/root/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 /DecidoDev/root/yii/framework/web/CWebApplication.php(282): CController->run('')
#15 /DecidoDev/root/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('user/install')
#16 /DecidoDev/root/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#17 /DecidoDev/root/testdrive/index.php(13): CApplication->run()
#18 {main}
REQUEST_URI=/testdrive/index.php?r=user/install
---
2013/01/24 15:58:47 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'testdrive'' in /DecidoDev/root/yii/framework/db/CDbConnection.php:381
Stack trace:
#0 /DecidoDev/root/yii/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /DecidoDev/root/yii/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /DecidoDev/root/yii/framework/base/CModule.php(387): CDbConnection->init()
#3 /DecidoDev/root/yii/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /DecidoDev/root/testdrive/protected/modules/user/controllers/YumInstallController.php(439): CModule->__get('db')
#5 /DecidoDev/root/yii/framework/web/actions/CInlineAction.php(49): YumInstallController->actionInstall()
#6 /DecidoDev/root/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#7 /DecidoDev/root/yii/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#8 /DecidoDev/root/yii/framework/web/filters/CFilter.php(40): CFilterChain->run()
#9 /DecidoDev/root/yii/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#10 /DecidoDev/root/yii/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#11 /DecidoDev/root/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#12 /DecidoDev/root/yii/framework/web/CController.php(291): CFilterChain->run()
#13 /DecidoDev/root/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 /DecidoDev/root/yii/framework/web/CWebApplication.php(282): CController->run('')
#15 /DecidoDev/root/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('user/install')
#16 /DecidoDev/root/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#17 /DecidoDev/root/testdrive/index.php(13): CApplication->run()
#18 {main}
REQUEST_URI=/testdrive/index.php?r=user/install
---
2013/01/24 15:59:01 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'testdrive'' in /DecidoDev/root/yii/framework/db/CDbConnection.php:381
Stack trace:
#0 /DecidoDev/root/yii/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /DecidoDev/root/yii/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /DecidoDev/root/yii/framework/base/CModule.php(387): CDbConnection->init()
#3 /DecidoDev/root/yii/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /DecidoDev/root/testdrive/protected/modules/user/controllers/YumInstallController.php(439): CModule->__get('db')
#5 /DecidoDev/root/yii/framework/web/actions/CInlineAction.php(49): YumInstallController->actionInstall()
#6 /DecidoDev/root/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#7 /DecidoDev/root/yii/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#8 /DecidoDev/root/yii/framework/web/filters/CFilter.php(40): CFilterChain->run()
#9 /DecidoDev/root/yii/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#10 /DecidoDev/root/yii/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#11 /DecidoDev/root/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#12 /DecidoDev/root/yii/framework/web/CController.php(291): CFilterChain->run()
#13 /DecidoDev/root/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 /DecidoDev/root/yii/framework/web/CWebApplication.php(282): CController->run('')
#15 /DecidoDev/root/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('user/install')
#16 /DecidoDev/root/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#17 /DecidoDev/root/testdrive/index.php(13): CApplication->run()
#18 {main}
REQUEST_URI=/testdrive/index.php?r=user/install
---
2013/01/24 16:02:12 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'testdrive.user' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `user`.
in /DecidoDev/root/testdrive/protected/modules/user/controllers/YumInstallController.php (439)
in /DecidoDev/root/testdrive/index.php (13)
2013/01/24 16:04:47 [error] [exception.CException] exception 'CException' with message 'Please make sure that Yii uses the YumWebUser component instead of CWebUser in your config/main.php components section. Please see the installation instructions.' in /DecidoDev/root/testdrive/protected/modules/user/controllers/YumUserController.php:98
Stack trace:
#0 /DecidoDev/root/yii/framework/web/CController.php(306): YumUserController->beforeAction(Object(CInlineAction))
#1 /DecidoDev/root/yii/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 /DecidoDev/root/yii/framework/web/filters/CFilter.php(40): CFilterChain->run()
#3 /DecidoDev/root/yii/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 /DecidoDev/root/yii/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 /DecidoDev/root/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 /DecidoDev/root/yii/framework/web/CController.php(291): CFilterChain->run()
#7 /DecidoDev/root/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 /DecidoDev/root/yii/framework/web/CWebApplication.php(282): CController->run('')
#9 /DecidoDev/root/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('user/login')
#10 /DecidoDev/root/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#11 /DecidoDev/root/testdrive/index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/testdrive/index.php?r=user/login
HTTP_REFERER=http://root.local/testdrive/index.php?r=user/install/install
---
2013/01/24 16:15:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "registration".' in /DecidoDev/root/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /DecidoDev/root/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('registration')
#1 /DecidoDev/root/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /DecidoDev/root/testdrive/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/testdrive/index.php?r=registration
---
2013/01/24 16:15:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "user/registration".' in /DecidoDev/root/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /DecidoDev/root/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('user/registrati...')
#1 /DecidoDev/root/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /DecidoDev/root/testdrive/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/testdrive/index.php?r=user/registration
---
2013/01/24 16:15:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "/user/registration".' in /DecidoDev/root/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /DecidoDev/root/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('/user/registrat...')
#1 /DecidoDev/root/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /DecidoDev/root/testdrive/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/testdrive/index.php?r=/user/registration
---
2013/01/24 16:23:06 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "registrate".' in /DecidoDev/root/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /DecidoDev/root/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('registrate')
#1 /DecidoDev/root/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /DecidoDev/root/testdrive/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/testdrive/index.php?r=registrate
---
2013/01/24 16:23:11 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "registration".' in /DecidoDev/root/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /DecidoDev/root/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('registration')
#1 /DecidoDev/root/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /DecidoDev/root/testdrive/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/testdrive/index.php?r=registration
---
2013/01/24 16:23:17 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "registration/user".' in /DecidoDev/root/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /DecidoDev/root/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('registration/us...')
#1 /DecidoDev/root/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /DecidoDev/root/testdrive/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/testdrive/index.php?r=registration/user
---
